packages/astro/src/core/build/plugins/plugin-ssr.ts

```diff
@@ -283,7 +283,7 @@ function generateSSRCode(settings: AstroSettings, adapter: AstroAdapter, middlew
 		`import { renderers } from '${RENDERERS_MODULE_ID}';`,
 		`import * as serverEntrypointModule from '${ADAPTER_VIRTUAL_MODULE_ID}';`,
 		`import { manifest as defaultManifest } from '${SSR_MANIFEST_VIRTUAL_MODULE_ID}';`,
-		edgeMiddleware ? `` : `import { onRequest as middleware } from '${middlewareId}';`,
+		// edgeMiddleware ? `` : `import { onRequest as middleware } from '${middlewareId}';`,
```

Comment:
This solves the error of not getting preview working at all, but now causes an error 500

Reply:
```suggestion
```